hobby.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang = "en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="description" content="Portfolio">
+        <meta name="viewport" content='width=device-width, inital-scale=1, shrink-to-fit=no'>
+        <title>Cadence</title>
+        <link rel="stylesheet" type="text/css" href="css/style.css">
+    
+    </head>
+    <body>
+       
+            <header>
+            <div id="myLinks">
+                <a class="headerLink" href="index.html">Home</a>
+                <a class="headerLink" href="about.html">About</a>
+                <a class="headerLink" href="portfolio.html">Portfolio</a>
+                <a class="headerLink" href="hobby.html">Hobby</a>
+            </div>
+        </header>
+        <section class="hobby-page">
+            <div class='hobby'>
+                <p>
+                    I love to living in San Francisco Bay Area, it locates in West Coast and I can view the beautiful Pacific Ocean. 
+                    I often go hiking on weekend to see the big ocean and relax myself. I often visit Mori Point Hiking Trail, 
+                    it has a very goo good view of Sunset. I can see a clear view. 
```

Comment:
Here are few grammar suggestions for you.
```suggestion
                <p>
    I love to live in San Francisco Bay Area, it's located on West Coast and I can view the beautiful Pacific Ocean. 
                    I often go hiking on the weekend to see the big ocean and relax. I often visit Mori Point Hiking Trail, 
                    it has a very good view of Sunset. I can see a clear view. ```